### DISCUSSION TITLE
Guess the song 🎸

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#786] [Galaxy Trivia] [answered]
```
E|-----------------|-----------------|
B|-----------------|-----------------|
G|-----------------|-----------------| x12
D|---------5-------|-----------------|
A|-7-----7----7--5-|-3-------2-------|
E|-----------------|-----------------|
```
**Bonus:** Name the songwriter.

**Comment by SANKARAMDAS** [comment, marked as answer]
The White Stripes – Seven Nation Army
Songwriter - Jack White

**Comment by peterbaccaro** [comment]
The tab is for the verse of the song **Seven Nation Army** by **The White Stripes**.  The song was written by **Jack White**, who is primarily known as the lead vocalist and guitarist of the American rock duo.
